tools/serverpod_cli/lib/src/create/create.dart

```diff
@@ -443,8 +448,13 @@ void _copyModuleTemplates(
       ),
       Replacement(
         slotName: 'VERSION',
-        replacement: templateVersion,
+        replacement: customServerpodPath == null ? templateVersion : '',
       ),
+      if (customServerpodPath != null)
+        Replacement(
+          slotName: 'path: ../../../packages/serverpod',
+          replacement: 'path: $customServerpodPath/packages/serverpod',
```

Comment:
Question: is this valid on windows?

Reply:
Yes, apparently windows handles a mix of Poxis and windows path in the pubspec file. This is just a copy paste of the solution that is implemented for the main serverpod project. We can clean this up later to get a bette file path :)